kong/cli/utils/services.lua

```diff
@@ -13,8 +13,8 @@ _M.STATUSES = {
 -- Services ordered by priority
 local services = {
   require "kong.cli.services.dnsmasq",
-  require "kong.cli.services.nginx",
-  require "kong.cli.services.serf"
+  require "kong.cli.services.serf",
+  require "kong.cli.services.nginx"
```

Comment:
> this would allow the older behavior to work again.

The old behavior being "run kong w/ Nginx `daemon off`" to get stderr/stdout streams?

It's an improvement indeed, as log spelunking on Docker containers makes ops difficult. Stream all the logs!